libs/api/protocol/src/lib/mapper-profile/protocol-entry.mapper.ts

```diff
@@ -53,6 +61,12 @@ export class ProtocolEntryMapper {
 					CommunicationMessageDocument,
 					CommunicationMessage,
 				);
+			case ProtocolEntryType.RESCUE_STATION_SIGN_ON_ENTRY:
```

Comment:
What about `RescueStationSignOffMessageDocument` and `RescueStationUpdateMessageDocument`?